src/qml/PluginManagerSettings.qml

```diff
@@ -301,17 +297,16 @@ Popup {
       }
     }
 
-    standardButtons: Dialog.Ok | Dialog.Cancel
     onAccepted: {
       pluginManager.installFromUrl(installFromUrlInput.text);
     }
   }
 
-  Dialog {
+  QfDialog {
     id: uninstallConfirmation
     title: "Uninstall Plugin"
     parent: mainWindow.contentItem
-    x: (mainWindow.width - width) / 2
+    modal: false
```

Comment:
I don't think we need modal: false here; the lack of model: true was most likely me being forgetful.

Reply:
Agreed.